app/code/Magento/Sales/Test/Mftf/Test/AdminInvoiceOrderTest.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminInvoiceOrderTest">
+        <annotations>
+            <features value="Sales"/>
+            <stories value="Create an Invoice via the Admin"/>
+            <title value="Admin should be able to create an invoice"/>
+            <description value="Admin should be able to create an invoice"/>
+            <severity value="MAJOR"/>
+            <testCaseId value="MAGETWO-72096"/>
+            <group value="sales"/>
+        </annotations>
+
+        <before>  
+            <createData entity="_defaultCategory" stepKey="createCategory"/>
+            <createData entity="_defaultProduct" stepKey="createSimpleProductApi">
+                <requiredEntity createDataKey="createCategory"/>
+            </createData>
+            <createData entity="GuestCart" stepKey="createGuestCart"/>
+            <createData entity="SimpleCartItem" stepKey="addCartItem">
+                <requiredEntity createDataKey="createGuestCart"/>
+                <requiredEntity createDataKey="createSimpleProductApi"/>
+            </createData>
+            <createData entity="GuestAddressInformation" stepKey="addGuestOrderAddress">
+                <requiredEntity createDataKey="createGuestCart"/>
+            </createData>
+            <updateData createDataKey="createGuestCart" entity="GuestOrderPaymentMethod" stepKey="sendGuestPaymentInformation">
+                <requiredEntity createDataKey="createGuestCart"/>
+            </updateData>   
+
+            <actionGroup ref="AdminLoginActionGroup" stepKey="loginAsAdmin"/>
+            
+        </before>
+
+        <after>
+            <deleteData createDataKey="createCategory" stepKey="deleteCategory"/>
+            <deleteData createDataKey="createSimpleProductApi" stepKey="deleteSimpleProductApi"/>
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="adminLogout"/>
```

Comment:
Ideally, we should remove all side effects created in the test. That means that it would be great to remove the `GuestCart` as well after the test, if possible.

Thanks

Reply:
@rogyar, I suggest there is no need/possibility to delete `GuestCart`after the test -  as we place the order during the test. In addition, there is no `operation` with `type="delete"` in the [GuestCartMeta.xml ](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Quote/Test/Mftf/Metadata/GuestCartMeta.xml) - probably because of the same logic.
Please, correct me if I'm mistaken.

Thanks.

Reply:
It's more or less what I was expecting. Just wanted to double-check, maybe we already have a routine for removing the quote. 

Nevermind then, thank you for checking.